### PR TITLE
REST API tests with skeletons

### PR DESCRIPTION
**Pull request by yasakova-anastasia**
### Motivation and context

Depends on https://github.com/opencv/cvat/pull/4867

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [x] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: tests/python/rest_api/test_projects.py [hunk 1]
```diff
@@ -349,10 +349,80 @@ def test_can_import_dataset_in_org(self, admin_user):
...
+            (_, response) = api_client.projects_api.create_backup(
+                backup_write_request=deepcopy(import_data),
+                _content_type="multipart/form-data")
+            assert response.status == HTTPStatus.ACCEPTED
+
 @pytest.mark.usefixtures('changedb')
 class TestPatchProjectLabel:
     def test_admin_can_delete_label(self, projects):
-        project = deepcopy(list(projects)[0])
+        project = deepcopy(list(projects)[1])
```
[the remark this answers is not in the document]
Comment (yasakova-anastasia) [reply tests/python/rest_api/test_projects.py:422]: I fixed this issue and added a test to remove skeleton label

**Review by kirill-sizov — APPROVED** [review]
LGTM

**Review by nmanovic — APPROVED** [review]
LGTM

**Pull request merged**